extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -255,4 +258,9 @@ HealthBuildItem addHealthCheck(KafkaBuildTimeConfig buildTimeConfig) {
         return new HealthBuildItem("io.quarkus.kafka.client.health.KafkaHealthCheck",
                 buildTimeConfig.healthEnabled);
     }
+
+    @BuildStep
+    UnremovableBeanBuildItem ensureJsonParserAvailable() {
+        return UnremovableBeanBuildItem.beanTypes(ObjectMapper.class, Jsonb.class);
```

Comment:
USe the string version of the class names, so you don't have to pull the dependencies.

Reply:
Isn't that risky in the case the classes ever get renamed or moved? Can this be detected by the compiler then?

Reply:
@cescoffier: I switched to using the string version as you suggested. When adding the deployment dependencies, the build didn't work anymore anyways. The current version worked for me.

Reply:
It's a bit more fragile but it's the only to not end up with a big dependency mess :).